gen/doc.jl

```diff
@@ -72,7 +71,74 @@ function build_docstring(node::EzXML.Node)
     end
 end
 
+"Return the text of a subelement `el` of `node`"
 function text(node::EzXML.Node, el::AbstractString)
     s = findfirst(el, node)
     s === nothing ? "" : strip(nodecontent(s))
 end
+
+"Wrap the one or multiline docstring in appropriate quotes"
```

Comment:
nit: "Wrap the single- or multi-line docstring in appropriate quotes."

Reply:
Thanks. Also switched to `repr` for single lines, since for PROJ at least I had an issue with unescaped quotes.